Week2/todo_app/program.js

```diff
@@ -62,11 +126,68 @@ dbConnection.connect(function(err) {
     console.log('connected as id ' + dbConnection.threadId);
 
     const todoModel = new TodoModel(dbConnection);
-    todoModel.load(function(err, todoItems) {
-        if(err) {
+    //Load all TODOs
+    todoModel.load(function (err, todoItems) {
+        if (err) {
             console.log("error loading TODO items:", err);
         }
+        console.log("existing todo items:", todoItems);
+    });
+
+    //create new TODO
+    todoModel.create(1, "Gaorieh Begin with week2", function (err, todoItems) {
```

Comment:
As an app, we'd like to get input from the user, instead of a demo with pre-set values. Can you try to make this happen?

Reply:
But here i see that we get info(parameters) from the user !!??

Reply:
we can do it with ( progress.argv )